### ISSUE TITLE
Test error on 0.4 release branch

### ISSUE THREAD

**Issue by eschnett** [JuliaLang/julia#15733]
After updating on the 0.4 release branch, I see this error in `make -j8testall` on Darwin:
```
Exception running test linalg/svd :
On worker 7:
LoadError: MethodError: `getindex` has no method matching getindex(::Base.RefValue{Int64}, ::Int64)
Closest candidates are:
  getindex(::Base.RefValue{T})
 in ggsvd3! at linalg/lapack.jl:1733
 in svdfact! at linalg/svd.jl:84
 in svdfact at linalg/svd.jl:88
 [inlined code] from /Users/eschnett/src/julia/test/linalg/svd.jl:49
 in anonymous at no file:0
 in include_string at loading.jl:282
 in include_from_node1 at ./loading.jl:323
 [inlined code] from util.jl:179
 in runtests at /Users/eschnett/src/julia/test/testdefs.jl:7
 in anonymous at /Users/eschnett/src/julia/test/runtests.jl:36
 in anonymous at multi.jl:923
 in run_work_thunk at multi.jl:661
 [inlined code] from multi.jl:923
 in anonymous at task.jl:63
while loading /Users/eschnett/src/julia/test/linalg/svd.jl, in expression starting on line 21
ERROR: LoadError: Some tests exited with errors.
 in anonymous at /Users/eschnett/src/julia/test/runtests.jl:64
 in cd at ./file.jl:22
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:320
 in process_options at ./client.jl:280
 in _start at ./client.jl:378
while loading /Users/eschnett/src/julia/test/runtests.jl, in expression starting on line 13
make[1]: *** [all] Error 1
make: *** [testall] Error 2
```

**Comment by yuyichao** [comment]
I believe this is due to https://github.com/JuliaLang/julia/pull/14444 and was fixed on master by https://github.com/JuliaLang/julia/pull/14522 . We can probably fix it on 0.4 with a smaller patch by replacing `info[1]` with `info[]`.

**Comment by andreasnoack** [comment]
Why haven't we caught this in CI?

**Comment by yuyichao** [comment]
Openblas version

**Comment by tkelman** [comment]
Bump @nalimilan your backport was buggy and it'll need to be fixed before we can do 0.4.6. Can you take a look?

**Comment by nalimilan** [comment]
Sorry for the slow reaction. https://github.com/JuliaLang/julia/pull/15883 fixes the failure with 0.2.16 in my tests (I wonder how I introduced it in the first place -- maybe I changed something right after testing).

**Comment by tkelman** [comment]
Should be fixed, but wouldn't mind a confirmation from anyone who was able to reproduce this.
